sklearn/utils/extmath.py

```diff
@@ -109,16 +109,6 @@ def fast_logdet(A):
     return ld
 
 
-def _impose_f_order(X):
-    """Helper Function"""
-    # important to access flags instead of calling np.isfortran,
-    # this catches corner cases.
-    if X.flags.c_contiguous:
-        return check_array(X.T, copy=False, order='F'), True
-    else:
-        return check_array(X, copy=False, order='F'), False
```

Comment:
Added 5 years ago in https://github.com/scikit-learn/scikit-learn/commit/2240e44f3195321d873dec4a9bc905186697088d and is currently not used in the code base